frame/staking/src/lib.rs

```diff
@@ -2878,45 +3046,38 @@ impl<T: Config> Pallet<T> {
 
 	/// Get all of the voters that are eligible for the npos election.
 	///
+	/// `voter_count` imposes an implicit cap on the number of voters returned; care should be taken
+	/// to ensure that it is accurate.
+	///
 	/// This will use all on-chain nominators, and all the validators will inject a self vote.
 	///
 	/// ### Slashing
 	///
 	/// All nominations that have been submitted before the last non-zero slash of the validator are
 	/// auto-chilled.
```

Comment:
Ther term "Nominations" is a bit vague to me here. I thought only the validator is chilled on the slash its its nominations are removed, but the nomintator themself is not chilled?